terraform/scripts/run_node_exporter.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+export DEBIAN_FRONTEND=noninteractive
+
+readonly VERSION=1.3.1
+
+wget "https://github.com/prometheus/node_exporter/releases/download/v$VERSION/node_exporter-$VERSION.linux-amd64.tar.gz"
+tar xvfz node_exporter-$VERSION.linux-amd64.tar.gz
+cd node_exporter-$VERSION.linux-amd64
+
+nohup ./node_exporter &
```

Comment:
seems it's not running anymore after `terraform apply` finished.
Check out why and fix it

Reply:
May be relevant
https://stackoverflow.com/a/51777995